src/euidSdk.ts

```diff
@@ -44,6 +44,9 @@ declare global {
     __euid: EUID | SDKSetup | undefined;
   }
 }
+export function assertEUID(sdk: typeof window.__euid): asserts sdk is EUID {
+  if (!(sdk instanceof EUID)) throw new Error('What? This is impossible!');
```

Comment:
Are we leaving this error message as-is?

Reply:
Ah - good catch, thanks.